src/services/initialTodos.tsx

```diff
@@ -0,0 +1,8 @@
+import { TODO } from '../types/todo';
+import todosFromServer from '../api/todos';
+import { getUserById } from './userById';
+
+export const initialTodos: TODO[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
```

Comment:
If `getUserById` involves asynchronous operations (e.g., fetching data from an API), it should be handled asynchronously. Consider refactoring this part to handle promises or use async/await to ensure that user data is correctly fetched and assigned.